app/assets/javascripts/friendships.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
Let's be professional with our codebase, kindly delete this empty file.

Reply:
- Kindly cover the requirements from your last reviewer.

Reply:
- Kindly cover the requirements from your last reviewer.